src/hearings/containers/hearing-actuals/hearing-actuals.component.html

```diff
@@ -0,0 +1,11 @@
+<exui-app-header></exui-app-header>
+<div class="govuk-width-container">
+  <div class="page">
+    <main id="content" role="main">
+      <a href="javascript:void(0)" (click)="onBack()" class="govuk-back-link">Back</a>
+      <router-outlet></router-outlet>
+      <button class="govuk-button" data-module="govuk-button" (click)="onSubmit()">Submit hearing details</button>
+    </main>
+  </div>
+</div>
+<exui-app-footer></exui-app-footer>
```

Comment:
if the parent folder can be renamed from hearing-actuals to actuals-hearing to get it consistent with other folders/jouneys naming convention.

Reply:
If do this, I'd also have to rename the components/files etc. Also it is aptly named, as it is "Hearing Actuals" and not "Actual Hearings"

Reply:
just the folder is fine, probably best to be actual-hearing(adjective+noun), like the other folder request-hearing(verb+noun) which is better to define a feature. as this is the root folder for the whole jouney, for the sub-components/files it's fine you could name it as hearing actuals, the 'AWAITING HEARING ACTUALS' is not accurate that's why they asked to change to to 'AWAITING HEARING DETAILS'